ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -192,6 +192,14 @@ objects:
         done
         # oh god the blood
         sleep 180
+        # can remove this check once a proper test is added to origin e2e
```

Comment:
Why not write the e2e test?

Reply:
I'm doing that now.  This ensures no breaking changes are merged b4 that test is in place (auth team is refactoring, configuring oauth, etc now).  However, with how long this has taken to merge, might-as-well wait for the e2e, as i'm finally able to get to it now.

Reply:
This is, however, a good indicator that a cluster is healthy, ready to execute the e2e suite.  Configuring the kubeadmin user requires a patch, takes a few minutes for the cluster to recover from that.  Might want to keep this check in.  (Similar to https://github.com/openshift/release/blob/90f98d30e531b2dbb39b9d29b9ca95cef29211b6/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L185)

Reply:
Why doesn't the installer handle those before handing off the cluster as "ready" ?

@wking

Reply:
I am just very suspicious of any bash in here ..

Reply:
Yeah, this feels like something that should be part of the installer, not post installer.  Or if it shouldn't, we should have a good reason not to (like we don't want the password in the clear, but the private key is in the clear, so it's kind of a wash).

Reply:
@wking I don't see how this could be part of the installer?
a) Installer would run `oc login` and that would require oc be available?
or
b) Installer would have to vendor origin code to run the equivalent of oc login?
maybe I'm missing something, clue me in please if so.  
I've opened a PR to add a smoke-4 test for kubeadmin login, instead:  https://github.com/openshift/origin/pull/21733